server/task_executor_pitr_cutover.go

```diff
@@ -76,6 +78,26 @@ func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.
 		zap.String("original_database", task.Database.Name),
 		zap.String("pitr_database", pitrDatabaseName),
 		zap.String("old_database", pitrOldDatabaseName))
+
+	log.Info("Appending new migration history record...")
+	m := &db.MigrationInfo{
+		ReleaseVersion: server.profile.Version,
+		Version:        common.DefaultMigrationVersion(),
+		Namespace:      task.Database.Name,
+		Database:       task.Database.Name,
+		Environment:    task.Database.Instance.Environment.Name,
+		Source:         db.MigrationSource(task.Database.Project.WorkflowType),
+		Type:           db.Baseline,
+		Description:    fmt.Sprintf("pitr: swap database %s and %s", task.Database.Name, pitrDatabaseName),
```

Comment:
It's not a swap from user's perspective.

"Restoring database %s to time %s"

Reply:
Currently, we cannot get the targetTs in this task.
"PITR: restoring database %s" for now.